# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../rrdtool/glib.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur rrdtool-1.4.4-orig/src/rrd_daemon.c rrdtool-1.4.4/src/rrd_daemon.c
--- rrdtool-1.4.4-orig/src/rrd_daemon.c	2010-09-07 12:25:57.909777061 +0000
+++ rrdtool-1.4.4/src/rrd_daemon.c	2010-09-07 12:26:27.358226536 +0000
@@ -109,7 +109,7 @@
 #include <libgen.h>
 #include <grp.h>
 
-#include <glib-2.0/glib.h>
+#include <glib.h>
 /* }}} */
 
 #define RRDD_LOG(severity, ...) \
